google-cloud-clients/google-cloud-spanner/src/main/java/com/google/cloud/spanner/admin/database/v1/stub/DatabaseAdminStubSettings.java

```diff
@@ -297,9 +297,7 @@ public String extractNextToken(ListDatabasesResponse payload) {
 
             @Override
             public Iterable<Database> extractResources(ListDatabasesResponse payload) {
-              return payload.getDatabasesList() != null
-                  ? payload.getDatabasesList()
-                  : ImmutableList.<Database>of();
+              return payload.getDatabasesList();
```

Comment:
Does this change behavior?

Reply:
No, protobufs are not null and this check is not necessary.
I will need to file a bug for gapic-generators to fix this semantics.

Reply:
Ugh.  If this is generated code, I'm afraid that we won't be able to check this in.  We'd need a generator change instead.

Reply:
@sduskis This doesnt change behavior. 

@snehashah16  This was done to protected against NPE in paged iteration. Some client like compute do return null instead of empty list. This was done to fix one of such bug [#3736](https://github.com/googleapis/google-cloud-java/issues/3736)